tools/nnyeah/nnyeah/FieldTransformations.cs

```diff
@@ -0,0 +1,59 @@
+using System;
+using System.Collections.Generic;
+using Mono.Cecil;
+using Mono.Cecil.Cil;
```

Comment:
Lets start using nullable in new tools.

Reply:
Agreed! 👍

Reply:
In this case it would make more sense to enable it in the csproj for all the files.

Reply:
Yep, that was my though, but I was going file by file.